virtual-env.ipynb

```diff
@@ -26,78 +31,116 @@
+    "3. <a id=\"conda-virt-env-setup-step-install-packages\"></a>Activate the Conda environment by running the following command:\n",
     "   ```sh\n",
-    "   conda install -n <environment name> <package> [<package> ...]\n",
+    "   conda activate <environment name>\n",
     "   ```\n",
-    "\n",
-    "   <br>For example, the following command installs the SciPy, pandas version 0.25.0, and TensorFlow version 1.13.1 packages for a \"myenv\" environment:\n",
+    "   \n",
+    "   <br>For example, the following command activates an environment named \"myenv\":\n",
     "   ```sh\n",
-    "   conda install -n myenv scipy pandas=0.25.0 tensorflow=1.13.1 python=3.7\n",
-    "   ```\n",
+    "   conda activate myenv\n",
+    "   ```   \n",
+    "   <br>\n",
     "\n",
+    "4. <a id=\"conda-virt-env-setup-step-install-packages\"></a>Once you activate your environment, you can use either PIP or Conda to install the necessary Python packages.<br> To do this, replace `<package>` with the name of the package you want to install, and optionally add `==<version>` for PIP or `=<version>` for Conda. You can specify multiple packages in the same command.<br>\n",
     "\n",
-    "4.  <a id=\"conda-virt-env-setup-step-export-env\"></a>Export your new virtual environment to an environment file in a platform data container by running the following command.\n",
-    "    Replace `<container name>` with the name of a platform data container, `<directory path>` with an optional relative container-directories path, and `<environment name>` with the name of the environment that you created:\n",
-    "    ```sh\n",
-    "    conda env export -n <environment name> > /v3io/<container name>[/<directory path>]/<environment name>.yaml\n",
-    "    ```\n",
-    "\n",
-    "    <br>It's recommended that you save the environment to a virtual-environments directory in your running-user home directory (**users/&lt;running user&gt;**).\n",
-    "    For example, the following command creates a **users/&lt;running user&gt;/virtual_env/myenv.yaml** file:\n",
-    "    ```sh\n",
-    "    conda env export -n myenv > /v3io/users/$V3IO_USERNAME/virtual_env/myenv.yaml\n",
-    "    ```\n",
-    "    You can use the `User` data mount to the running-user directory to shorten this command (see [Platform Data Containers](data-ingestion-and-preparation/README.ipynb#platform-data-containers)):\n",
-    "    ```sh\n",
-    "    conda env -n myenv export > /User/virtual_env/myenv.yaml\n",
-    "    ```\n",
+    "   Using PIP:\n",
+    "   ```sh\n",
+    "   pip install <package> [<package> ...]\n",
+    "   ```\n",
+    "   <br>Alternatively, using Conda:\n",
+    "   ```sh\n",
+    "   conda install <package> [<package> ...]\n",
+    "   ```\n",
+    "   \n",
+    "   <br>For example, the following command uses PIP to install the SciPy, pandas version 1.4.4, and TensorFlow version 2.9.3 packages for the \"myenv\" environment that we activated in the previous step:\n",
```

Comment:
"   <br>For example, the following command uses PIP to install the SciPy, pandas version 1.4.4, and TensorFlow version 2.9.3 packages for the \"myenv\" environment that you activated in the previous step:\n",
    "   ```sh\n",